agent/main_test.go

```diff
@@ -139,13 +140,6 @@ func TestImports(t *testing.T) {
 		constraints[a] = c
 	}
 
-	// slowlog parser should be fully independent
-	constraints["github.com/percona/pmm/agent/agents/mysql/slowlog/parser"] = constraint{
-		denyPrefixes: []string{
-			"github.com/percona/pmm/agent",
-		},
-	}
-
```

Comment:
As discussed on 1to1 with Nurlan slowlog now can use utils. Because of filereader which is now shared.